geonode/settings.py

```diff
@@ -21,62 +21,75 @@
 # Django settings for the GeoNode project.
 import os
 from kombu import Queue
-import geonode
+from geonode import __file__ as geonode_path
+from geonode import get_version
 from geonode.celery_app import app  # flake8: noqa
+import djcelery
+import dj_database_url
+
+
+def str2bool(v):
```

Comment:
I ended up importing another implementation of `strtobool` from distutils, that way we don't have to reinvent the wheel.